README.md

```diff
@@ -1,5 +1,10 @@
 # RSKafka
 
+[![CircleCI](https://circleci.com/gh/influxdata/rskafka/tree/main.svg?style=shield&circle-token=531ba1f38035a10da6dbf7cc71e6f55eff496c70)](https://circleci.com/gh/influxdata/rskafka/tree/main)
```

Comment:
I case you're wondering: this is a status-only token (i.e. it can only read the status) which is required even though the project is public. It seems that w/o the token the GitHub image anonymizer (see <https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-anonymized-urls>) seems to run into rate limits or something resulting in a 404 when a user watches this README.